src/css/_youtube.scss

```diff
@@ -107,17 +107,16 @@
     text-shadow: 0 0 36px rgba(51, 51, 51, 0.4);
   }
   &--logo{
-    font-family: "Guardian Titlepiece Web";
-    font-size: 22px;
-    font-weight: 400;
+    width: 250px;
+    height: 121px;
+    overflow: hidden;
+    text-indent: -9999em;
```

Comment:
Worth checking with the seo team if there is a cleaner way to do this as this feels super hacky.